RecoTracker/IterativeTracking/python/iterativeTkConfig.py

```diff
@@ -80,6 +93,16 @@
     minNumberOfLayersWithMeasBeforeFiltering = 0,
 )
 
+#Phase2 : configuring the phase2 track Cluster Remover
+from RecoLocalTracker.SubCollectionProducers.phase2trackClusterRemover_cfi import *
+__trackClusterRemoverBase_Phase2PU140 = phase2trackClusterRemover.clone(
```

Comment:
In addition, use `trackingPhase2PU140` (i.e. the full sub-era name) as the postfix.